vhdbuilder/packer/install-dependencies.sh

```diff
@@ -205,6 +205,28 @@ EOF
 
 echo "${CONTAINER_RUNTIME} images pre-pulled:" >> ${VHD_LOGS_FILEPATH}
 
+installAndConfigureArtifactStreaming() {
+  # download acr-mirror proxy
+  MIRROR_PROXY_VERSION='19'
+  UBUNTU_VERSION_CLEANED="${UBUNTU_RELEASE//.}"
+  MIRROR_PROXY_URL="https://acrmirrordev.blob.core.windows.net/bin/Release-${MIRROR_PROXY_VERSION}/acr-mirror-${UBUNTU_VERSION_CLEANED}.deb"
+  
+  wget $MIRROR_PROXY_URL || exit $ERR_ARTIFACT_STREAMING_DOWNLOAD_INSTALL
+  apt_get_install 30 1 600 "./acr-mirror-${UBUNTU_VERSION_CLEANED}.deb" || exit $ERR_ARTIFACT_STREAMING_DOWNLOAD_INSTALL
+  systemctl stop acr-mirror.service
+
+  echo "  - [installed] acr mirror-proxy v${MIRROR_PROXY_VERSION}" >> ${VHD_LOGS_FILEPATH}
+  rm "./acr-mirror-${UBUNTU_VERSION_CLEANED}.deb"
+
+  sudo apt install libnl-3-dev libnl-genl-3-dev libc6 libssl3 -y
+
+  echo "  - [installed] libnl-3-dev, libnl-genl-3-dev, libc6 and libssl3" >> ${VHD_LOGS_FILEPATH}
+}
+
+if [[ ( "${UBUNTU_RELEASE}" == "20.04" || "${UBUNTU_RELEASE}" == "22.04" ) && ( "${CPU_ARCH}" == "amd64" ) ]]; then
```

Comment:
might be worth saving yourself future pain and splitting the major/minor, and doing >= 20 on the version.

Reply:
Updated. May add 18.04 support for FIPS depending on usage and will add Mariner support later.